### DISCUSSION TITLE
Deep link product from website for Fluxstore

### DISCUSSION THREAD

**Discussion by jsaristi** [inspireui/support#5539] [Working on Features]
Taking advantage of the existing functionality of firebase dynamic links and the mstore API wordpress plugin, I propose that you could place a smart banner like this (1 screenshot) that leads from the product on the web page to the product in the application. This functionality could create the dynamic links through the API from the WordPress plugin itself to the firebase API and placing a smart banner with the personalized link to each of the products to take it directly to the application (Open in App) the same for the post of the blog.

This would give more importance to the application.

![Captura de Pantalla 2022-05-29 a la(s) 6 46 06 p m](https://user-images.githubusercontent.com/29686082/170968425-a591418d-b217-4295-9c5c-56269421f10c.png)

**Comment by osman-kocak** [comment]
it really should be

**Comment by tryabp** [comment]
yes much needed

**Comment by Mokwa123** [comment]
Really need this please

**Comment by Uhtredtb** [comment]
Make it happens, please 🙏
